tests/test_search.py

```diff
@@ -19,3 +22,47 @@ def test_empty_response(client) -> None:
 
     assert validator.validate(xml) is True, validator.error_log
     assert root.getchildren() == []
+
+
+def test_search(client, mocker):
+    es_search = mocker.patch('search.search.Elasticsearch.search')
+    es_search.return_value = {
```

Comment:
just noting down: we stay at the unit testing level, whereas the need for integration tests using ElastiSearch will come up when we have more complex inputs like a query, filters or pages.